staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -173,6 +207,12 @@ type StatefulSetSpec struct {
 	// consists of all revisions not represented by a currently applied
 	// StatefulSetSpec version. The default value is 10.
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty" protobuf:"varint,8,opt,name=revisionHistoryLimit"`
+
+	// PersistentVolumeClaimPolicy describes the policy used for PVCs created from
+	// the StatefulSet VolumeClaimTemplates. This requires the
```

Comment:
Should be `persistentVolumeClaimPolicy` and `volumeClaimTemplates` - public facing godoc is described in terms of external fields, not internal fields.

I would expect the second sentence to describe the default behavior in brief (so a net new second sentence), with the detail being covered by the struct's fields.  I.e. use the field here to describe the high level behavior, and leave the details to the field.  The first sentence reads a little rough.  An improved example might be:

> persistentVolumeClaimPolicy describes the lifecycle of persistent volume claims created from volumeClaimTemplates.  By default, all persistent volume claims are created as needed and retained until manually deleted by user action. This policy allows you to alter the default lifecycle behavior. ...

Reply:
done. LMK if it needs further refinement.